agents/scsi/fence_scsi.py

```diff
@@ -375,6 +386,19 @@ def define_new_opts():
 		"default" : "@VGS_PATH@",
 		"order": 300
 	}
+	all_opt["key_value"] = {
+		"getopt" : ":",
+		"longopt" : "key_value",
```

Comment:
I would set this to `key-value`.

Reply:
when I have used the `key-value` then the `fence_scsi -o metadata` generates automatically `key_value` option as well with 'deprecated=1' option on CentOS 7.7 as seen below.
~~~
fence_scsi -o metadata
...
	<parameter name="key_value" unique="0" required="0" obsoletes="key-value">
...
	<parameter name="key-value" unique="0" required="0" deprecated="1">
...
~~~
I was trying to avoid using name that would result in this situation by using underscore. But there is no problem changing this. Just to confirm, you prefer changing this to `key-value` even if it will create above situation?

Reply:
Just update the CLI parameter (longopt and help fields) and leave the ["key_update"] as is.

Remember to also update where it's used search for `["--key`.

Reply:
commit ee7a5ea238b4b3312384e4cfd9edd392c311d17a